.github/workflows/check.yml

```
@@ -0,0 +1,30 @@
+on:
+  pull_request:
+    branches:
+      - '*'
+
+name: Project check
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
```

```suggestion
          node-version: '24'
```
Давай подрубим 24, позавчера вышла